test/verify/check-setroubleshoot

```diff
@@ -114,7 +114,7 @@ class TestSelinux(MachineCase):
 
         row_selector = "tbody:contains('sshd from open access on the file')"
 
-        if self.machine.image in ["rhel-7-4"]:
+        if self.machine.image in ["rhel-7", "rhel-7-4"]:
```

Comment:
Yeah, agreed; "setroubleshoot too old" is either wrong or at least not telling the whole story - there is no violation with our test in the first place. I'm not SELinux savy enough to figure out how to get a violation, though.